renderer/brave_wallet/brave_wallet_render_frame_observer.cc

```diff
@@ -11,6 +11,8 @@
 #include "base/feature_list.h"
 #include "brave/components/brave_wallet/common/features.h"
 #include "brave/components/brave_wallet/renderer/v8_helper.h"
```

Comment:
#include "build/buildflag.h"

Reply:
updated, as above, why the system doesn't pin point if a required header file should be added/missing?

Reply:
Asked it here https://github.com/brave/brave-core/pull/15979#discussion_r1025639750